tracing-subscriber/src/fmt/format/mod.rs

```diff
@@ -673,7 +695,16 @@ where
         let bold = self.bold();
         let mut seen = false;
 
-        self.ctx.visit_spans(|span| {
+        let span = self
+            .span
+            .and_then(|id| self.ctx.ctx.span(&id))
+            .or_else(|| self.ctx.ctx.lookup_current());
```

Comment:
This is needed because this an option and we don't have stable try blocks, right?

Reply:
What do you mean? If there is no explicitly set parent, we want to look at the context to determine the current span.